.github/workflows/pythonbuild.yml

```diff
@@ -205,9 +205,9 @@ jobs:
             tox-env: py312-azureblob
 
 
-          - python-version: 3.9
+          - python-version: "3.10"
             tox-env: flake8
-          - python-version: 3.9
+          - python-version: "3.10"
```

Comment:
use python 3.10 because docs build failed on 3.9
python for flake 8 also updated as a represented environment